src/components/mumble-detail.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import type { FC } from "react";
+import {
+  IconButton,
+  MumbleActions,
+  MumbleDetailView,
+  ProfileIcon,
+} from "@ost-cas-fea-25-26/pp-design-system";
+import { Post } from "@/lib/api/posts/posts.types";
+import { createReplyForPostAction } from "@/lib/actions/posts.actions";
+import { MumbleUser } from "@/lib/mappers/user.mappers";
+
+type MumbleDetailTypeProps = {
+  mumble: Post;
+  author: MumbleUser;
+  replies: Post[];
+  deepLink: string;
+  currentUser: MumbleUser;
+};
+
+export const MumbleDetail: FC<MumbleDetailTypeProps> = ({
+  mumble,
+  author,
+  currentUser,
+  replies,
+  deepLink,
+}) => {
+  if (!mumble.id) {
+    throw new Error("Mumble id is required");
```

Comment:
I wouldn’t throw errors inside a component. It’s better to render an explicit error component instead.
That said, for the sake of speed, it’s totally fine to leave a TODO comment here for now 😊

(The same applies to all other places in this PR where errors are thrown at the component level.)